src/read/line.rs

```diff
@@ -575,6 +574,49 @@ fn line_target_yaw(i: &str) -> IResult<VectorialStrafingConstraints> {
                 tolerance: tolerance.unwrap_or(0.),
             },
         ),
+        map(
+            preceded(
+                tag("look_at"),
+                preceded(
```

Comment:
Sounds like you should split this entire thing into a subfunction (`parse_look_at`) and also probably wrap it in a `cut()` so that if we get a `look_at` then we must parse the rest of the line, instead of trying another parser.